c/charsets.c

```diff
@@ -199,23 +199,35 @@ int convertCharset(char *input,
   }
 }
 
-int getCharsetCode(char *charsetName) {
-  if (!strcmp(charsetName, "ISO-8859-1")) {
+#define CHARSETNAME_SIZE 15
+int getCharsetCode(const char *charsetName) {
```

Comment:
@sbollingerAtRocket please correct the declaration too.

Reply:
@1000TurquoisePogs @sbollingerAtRocket this hasn't been addressed.

Reply:
What part are you referring.  I created a local array with an additional character at the end and have it initialized to guarantee that the last element is null.  The code then test to verify the the input name will not overwrite it before coping it into the local array which then the code does a convert to upper case.

Reply:
> @sbollingerAtRocket please correct the declaration too.